STL_Extension/include/CGAL/Uncertain.h

```diff
@@ -67,7 +67,7 @@ class Uncertain_conversion_exception
   Uncertain_conversion_exception(const std::string &s)
     : std::range_error(s) {}
 
-  ~Uncertain_conversion_exception() throw() {}
+  ~Uncertain_conversion_exception() noexcept {}
```

Comment:
Isn't `noexcept` the default for destructors? So we could remove `throw()` there instead of replacing it with `noexcept`. Quite possibly we could even remove the destructor...

Reply:
Indeed.

https://en.cppreference.com/w/cpp/language/destructor
> if no exception specification is explicitly provided, the exception specification is considered to be one that would be used by the implicitly-declared destructor. In most cases, this is noexcept(true).

Reply:
Sorry I have been a bit busy with my GSoC project at Inkscape, and as it's hosted on GitLab I rarely visit GH.
Will do this soon :heart: